odl/util/testutils.py

```diff
@@ -64,32 +66,14 @@ def dtype_places(dtype, default=None):
         return default if default is not None else 5
 
 
-def almost_equal(a, b, places=None):
-    """Return ``True`` if the scalars ``a`` and ``b`` are almost equal."""
-    if a is None and b is None:
-        return True
-
-    if places is None:
-        places = _places(a, b)
-
-    eps = 10 ** -places
-
-    try:
-        complex(a)
-        complex(b)
-    except TypeError:
-        return False
-
-    if np.isnan(a) and np.isnan(b):
-        return True
-
-    if np.isinf(a) and np.isinf(b):
-        return a == b
+def dtype_rtol(dtype, default=None):
+    """Return a relative tolerance for a given dtype.
```

Comment:
No parameter list for this function?

Reply:
Hm yeah, I guess I was just too lazy :-)

Reply:
I removed the `r` from the function name since the tolerance can also be used as absolute. It also has a bit of doc, not the full `Parameters` section though. Should be good enough for an internal util.